assembl/static2/js/app/components/administration/landingPage/selectModulesForm.jsx

```diff
@@ -21,9 +21,8 @@ export const DumbSelectModulesForm = ({ hasErrors, modulesByIdentifier, moduleTy
           return (
             <Checkbox
               key={moduleType.identifier}
-              checked={module && module.get('enabled')}
-              onChange={() => toggleModule(moduleType.identifier)}
-              disabled={moduleType.required}
```

Comment:
you should probably keep this disabled line

Reply:
Roxana wants the required checkbox to be in the same style as the other. Should I change the checkbox required style with CSS rather than changing the js file ?